helm-charts/karpenter-resources/values.yaml

```diff
@@ -16,8 +16,8 @@ ec2NodeClass:
     httpProtocolIPv6: disabled
     httpPutResponseHopLimit: 2
     httpTokens: required
-  blockDevice:
-    deviceName: /dev/xvda
+  blockDeviceMappings:
+  - deviceName: /dev/xvda
     volumeSize: 100Gi
```

Comment:
Do you need to add `ebs:` here? Check the docs here https://karpenter.sh/docs/concepts/nodeclasses/#specblockdevicemappings

Reply:
Yes, I should have added ebs: here. My local test cases only covers overwriting the default values